m0t1fs/linux_kernel/st/m0t1fs_fsync_test.sh

```diff
@@ -19,15 +19,15 @@
 #
 
 
-. `dirname $0`/common.sh
-. `dirname $0`/m0t1fs_common_inc.sh
-. `dirname $0`/m0t1fs_client_inc.sh
-. `dirname $0`/m0t1fs_server_inc.sh
-. `dirname $0`/m0t1fs_sns_common_inc.sh
+. `dirname "$0"`/common.sh
```

Comment:
Please replace the backticked with $()
Please see https://github.com/Seagate/cortx-motr/pull/1857

Reply:
done